README.md

```diff
@@ -71,7 +71,7 @@ In order to remove the created folders from your gh-pages, you can use either:
 - `grunt shell:remove_folder --keep --folder=br_branchname1,br_branchname2,...`: only keeps the specified folder(s) on your gh-pages and removes everything else. Just add the `--keep` flag.
 
 ### Preview on your local machine
-- To preview your changes locally, run `sudo grunt serve`
+- To preview your changes locally, run `grunt start`
```

Comment:
`start` command is for the first run of the project, it's not necessary for every change.

Reply:
Oh okay, will update it

Reply:
the common misconception is that most people thought grunt serve will get their change before they run the command, meanwhile grunt serve is previewing the change after it runs the command. most people get used to npm start concept, where it bulid and serve your app. 

moreover, serve is only serving js, an css changes , jsx or adding new files they should run grunt shell:compile_dev and grunt copy respectively.

let's make PR to revamp binary readme to mention these in detail @amirghadimi-fs @khalidxx

Reply:
I have a PR for binary-static [here](https://github.com/binary-com/binary-static/pull/6386). I'll make for SmartTrader as well